integration/command.go

```diff
@@ -85,7 +85,7 @@ func DeployInspektorGadget(image, imagePullPolicy string) *Command {
 	return &Command{
 		Name:           "DeployInspektorGadget",
 		Cmd:            cmd,
-		ExpectedRegexp: `\d\/\d gadget pod\(s\) ready`,
+		ExpectedRegexp: `\d\/\d gadget pod\(s\) ready|The gadget pod\(s\) weren't modified!`,
```

Comment:
I think we should not rely on the return value because it is also 0 when the IG is already running, and we want the test to fail in that case, see [my comment](https://github.com/inspektor-gadget/inspektor-gadget/pull/1058#pullrequestreview-1149021120).

However, you are right. The regex is not correct. I think we should do:

```suggestion
		ExpectedRegexp: "Inspektor Gadget successfully deployed",
```

Reply:
Yep. As I said [my comment](https://github.com/inspektor-gadget/inspektor-gadget/pull/1058#pullrequestreview-1149021120), I don't think we should introduce the change that Burak is proposing. 

However, the current implementation needs to be updated to cover multi-node clusters, as you pointed out.